src/utils/warmup.rs

```diff
@@ -0,0 +1,139 @@
+//! Provides functions for initializing and "warming up" the database, such as setting snap categories.
+#![allow(dead_code, unused_imports)]
```

Comment:
Do we still need this?

Reply:
Yes, because in some compilation targets it's not used 🙃